--- config.def.h.old	2020-06-19 18:29:45.000000000 +0900
+++ config.def.h	2022-01-02 22:37:31.254889558 +0900
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "VL Gothic:pixelsize=22:antialias=true:autohint=true";
 static int borderpx = 2;
 
 /*
@@ -100,7 +100,7 @@
 	"red3",
 	"green3",
 	"yellow3",
-	"blue2",
+	"blue3",
 	"magenta3",
 	"cyan3",
 	"gray90",
@@ -110,7 +110,7 @@
 	"red",
 	"green",
 	"yellow",
-	"#5c5cff",
+	"#729fcf",
 	"magenta",
 	"cyan",
 	"white",
